libs/sdk-flutter/lib/native_toolkit.dart

```diff
@@ -4,12 +4,15 @@ import 'bridge_generated.dart';
 
 BreezSdkCore? _breezSDK;
 
+const _libName = "libbreez_sdk_core.so";
+
 BreezSdkCore getNativeToolkit() {
   if (_breezSDK == null) {
-    final DynamicLibrary lib = Platform.isAndroid
-      ? DynamicLibrary.open("libbreez_sdk_core.so")   // Load the dynamic library on Android
-      : DynamicLibrary.process();
+    final DynamicLibrary lib = Platform.isIOS
+        ? DynamicLibrary.process() // iOS is statically linked
+        : DynamicLibrary.open(
+            Platform.isAndroid ? _libName : "breez-sdk/libs/sdk-flutter/linux/lib/$_libName");
```

Comment:
Why is this path used and not just `_libName` ?

Reply:
I was being lazy and hard-coded where it ended up in my project. The correct way is to set `LD_LIBRARY_PATH` env variable. I've added a comment to reflect that.